src/shared/libs/constants.ts

```diff
@@ -0,0 +1,5 @@
+export const NULLIFY_UUID_VALUE: string =
+  "ffffffff-ffff-ffff-ffff-ffffffffffff";
+
+export const uuidRegex =
```

Comment:
We probably should have all these constants in upper case.
Also we can apply some grouping in names, like GROUP_VALUE, e.g. REGEX_UUID (we can later add REGEX_EMAIL, REGEX_PHONE_NUMBER, etc.)
Alternatively we can make constant objects like this:
```
export const REGEX_PATTERNS = {
  UUID: /^[a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12}$/
}
```